src/test/java/it/pagopa/transactions/commands/handlers/v1/TransactionRequestAuthorizationHandlerTest.java

```diff
@@ -906,149 +914,4 @@ void shouldMapLogoSuccessfully(String brand) {
         cardsTested = true;
         testedCardBrands.add(CardAuthRequestDetailsDto.BrandEnum.fromValue(brand));
     }
-
-    @Test
-    void shouldSaveAuthorizationEventGdiVerificationForWallet() {
```

Comment:
Note to reviewers: this test was removed because `v1` transactions don't support the authenticated flow, which was tested here (with `IO` client id)